src/style.scss

```diff
@@ -19,17 +19,27 @@
 // }
 
 
-// Selector is for initial setup only
-// delete when building new custom element
-.testClass {
+.toast {
   display: inline-block;
-  padding: var(--auro-text-body-size-default);
-  border: 1px solid var(--auro-color-border-error-on-light);
-  color:  var(--auro-color-border-error-on-light);
+  min-width: 100%;
+  min-height: 60px;
+  padding: var(--auro-size-200) var(--auro-size-150);
+  border-radius: 8px;
+  box-shadow: var(--auro-elevation-200);
+  
+  @media (576px <= width <= 1023px) {
+    min-width: 33%;
+    max-width: 50%;
+  }
+  @media (width >= 1024px) {
+    min-width: 200px;
+    max-width: 500px;
+  }
+
```

Comment:
These break points are curious to me. 

1. Why not use the WCSS provided breakpoints? 
2. Please use Auro Tokens when available. 

| token | value|
|---|---|
|var(--auro-grid-breakpoint-xs) | 320px |  
|var(--auro-grid-breakpoint-sm) | 576px |  
|var(--auro-grid-breakpoint-md) | 768px |  
|var(--auro-grid-breakpoint-lg) | 1024px |  
|var(--auro-grid-breakpoint-xl) | 1232px

Reply:
ohh i see, i will make sure to update that. i don't think i was using the WCSS breakpoints correctly so that's why i didn't push up those changes. but i just figured out how to reference them correctly in the scss file